engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/StructsLibrary.java

```diff
@@ -0,0 +1,27 @@
+package org.enso.interpreter.runtime.callable.atom;
+
+import com.oracle.truffle.api.library.GenerateLibrary;
+import com.oracle.truffle.api.library.Library;
+import com.oracle.truffle.api.library.LibraryFactory;
+
+@GenerateLibrary
+public abstract class StructsLibrary extends Library {
```

Comment:
I've heard `Library` isn't the most efficient (in terms of AST size) concept for use inside of a language, but I think we are not that far to switch to other concepts yet.

Reply:
Didn't Graal Team also mention that they want to eventually get rid of it as well?
I hope there will be an alternative.